i18n/vi.yaml

```diff
@@ -0,0 +1,25 @@
+- id: prev_page
+  translation: "Trang trước"
+
+- id: next_page
+  translation: "Trang tiếp theo"
+
+- id: read_time
+  translation:
+    one : "1 phút"
+    other: "{{ .Đếm }} phút"
```

Comment:
Here `.Count` is a hugo variable, need not be translated!

Can you add `.Count` in place of `.Đếm`

Reply:
Okay. Thank u